tests/functional/lib/helpers.js

```diff
@@ -203,7 +215,7 @@ define([
 
   function noSuchElement(context, selector) {
     return function () {
-      return context.remote
+      return this.parent
```

Comment:
It kind of sort of is, I want to move away from passing `context` and use `this.parent` everywhere. We always passed `context` because we didn't know about `this.parent` when these functions are used as promise callbacks. If I continue with this PR, I'll change `this.parent` to `getRemote(context)` to be consistent with the other changes.